src/test/java/org/apache/sysds/test/functions/federated/algorithms/FederatedLmPipeline.java

```diff
@@ -44,20 +46,27 @@
 	@Override
 	public void setUp() {
 		TestUtils.clearAssertionInformation();
-		addTestConfiguration(TEST_NAME, new TestConfiguration(TEST_CLASS_DIR, TEST_NAME, new String[] {"Z"}));
+		addTestConfiguration(TEST_NAME1, new TestConfiguration(TEST_CLASS_DIR, TEST_NAME1, new String[] {"Z"}));
+		addTestConfiguration(TEST_NAME2, new TestConfiguration(TEST_CLASS_DIR, TEST_NAME2, new String[] {"Z"}));
 	}
 
 	@Test
 	public void federatedLmPipelineContinguous() {
-		federatedLmPipeline(Types.ExecMode.SINGLE_NODE, true);
+		federatedLmPipeline(Types.ExecMode.SINGLE_NODE, true, TEST_NAME1);
 	}
+
+	@Test
+	public void federatedLmPipelineContinguous4Workers() { federatedLmPipeline(Types.ExecMode.SINGLE_NODE, true, TEST_NAME2); }
 	
 	@Test
 	public void federatedLmPipelineSampled() {
-		federatedLmPipeline(Types.ExecMode.SINGLE_NODE, false);
+		federatedLmPipeline(Types.ExecMode.SINGLE_NODE, false, TEST_NAME1);
 	}
 
-	public void federatedLmPipeline(ExecMode execMode, boolean contSplits) {
+	@Test
+	public void federatedLmPipelineSampled4Workers() { federatedLmPipeline(Types.ExecMode.SINGLE_NODE, false, TEST_NAME2); }
```

Comment:
i personally like it in one line as well, but our other tests, does not follow this syntax so i would suggest multi line.